frigate/stats.py

```diff
@@ -121,6 +121,15 @@ async def set_gpu_stats(config: FrigateConfig, all_stats: dict[str, Any]) -> Non
 
         if args and args not in hwaccel_args:
             hwaccel_args.append(args)
+     
+    for stream_input in camera.ffmpeg.inputs:
+        args = stream_input.hwaccel_args
+
+        if isinstance(args, list):
+            args = " ".join(args)
+
+        if args and args not in hwaccel_args:
+            hwaccel_args.append(args)
```

Comment:
```suggestion
     
        for stream_input in camera.ffmpeg.inputs:
            args = stream_input.hwaccel_args

            if isinstance(args, list):
                args = " ".join(args)

            if args and args not in hwaccel_args:
                hwaccel_args.append(args)
```

Reply:
If it's at the same level as the first loop it's only going to check the last camera where you want it to check all cameras.

Reply:
haha thanks copy paste fail